homeassistant/helpers/collection.py

```diff
@@ -375,6 +375,7 @@ def async_setup(self, hass: HomeAssistant, *, create_list: bool = True) -> None:
                 {
                     **self.create_schema,
                     vol.Required("type"): f"{self.api_prefix}/create",
+                    vol.Optional(self.item_id_key): str,
```

Comment:
This shouldn't be necessary, we can just add it to the create schema that we pass in ?

Reply:
fixed